.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Are we okay with daily or should we change to weekly?

Reply:
I'm fine with daily here, since there aren't that many dependencies (famous last words).